sources/packages/backend/apps/api/src/route-controllers/announcements/_tests_/e2e/announcement.institutions.controller.getAnnouncements.e2e-spec.ts

```diff
@@ -0,0 +1,127 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import { Repository } from "typeorm";
+import { Announcement } from "@sims/sims-db";
+import { addDays } from "@sims/utilities";
+import {
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getInstitutionToken,
+  InstitutionTokenTypes,
+} from "../../../../testHelpers";
+
+describe("AnnouncementInstitutionsController(e2e)-getAnnouncements", () => {
+  let app: INestApplication;
+  let announcementsRepo: Repository<Announcement>;
+  const endpoint = `/institutions/announcements?target=institution-dashboard`;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    announcementsRepo = dataSource.getRepository(Announcement);
```

Comment:
We have an object called `E2EDataSources` and `createE2EDataSources()` and we can add the announcement repo their to have easy access for this and future tests. 

As a reference, we can follow this code [Other E2E Test example](https://github.com/bcgov/SIMS/blob/main/sources/packages/backend/apps/api/src/route-controllers/report/_tests_/e2e/report.aest.controller.exportReport.e2e-spec.ts)

This comment is not a blocker. If you do not want to make the change, feel free to ignore.